api-auto-registration/installation.hbs.md

```diff
@@ -2,20 +2,18 @@
 
 This document describes how to install API Auto Registration from the Tanzu Application Platform package repository.
 
->**Note:** Use the instructions on this page if you do not want to use the iterate, run, or full profile to install packages.
-The "iterate", "run", and "full" profiles include API Auto Registration by default.
+>**Note:** The "iterate", "run", and "full" profiles include API Auto Registration by default. 
+> If your cluster is one of these profiles, skip the install and proceed to the [Usage section](usage.md).
```

Comment:
For `run` profile clusters, their installation is probably not going to work unless they specified `api_auto_registration.tap_gui_url` and `cluster_name` in their `tap-values.yaml`, so I'm worried about us telling users to skip the section that talks about values altogether. However, I'm not sure what's the best way to point that out.

Reply:
Same as previous. I will try to find the pattern that other teams follow. I think we have 3 scenarios, 1. full/iterate profile only optionally need to know about updates. 2. run requires the update. 3 all other need all the instructions. I may divide the instructions into sub sections to easily link to the concerned section for all these profiles.

Reply:
Check my newer commit's usage page first paragraph